mappings/net/minecraft/world/storage/WorldSaveStorage.mapping

```diff
@@ -85,19 +89,19 @@ CLASS net/minecraft/unmapped/C_bpwagnit net/minecraft/world/level/storage/LevelS
 			METHOD visitFile visitFile (Ljava/lang/Object;Ljava/nio/file/attribute/BasicFileAttributes;)Ljava/nio/file/FileVisitResult;
 		CLASS C_zvjetysm
 			METHOD visitFile visitFile (Ljava/lang/Object;Ljava/nio/file/attribute/BasicFileAttributes;)Ljava/nio/file/FileVisitResult;
-	CLASS C_wxntpspm LevelDirectory
+	CLASS C_wxntpspm WorldDirectory
 		FIELD f_iapirmxc path Ljava/nio/file/Path;
 		METHOD <init> (Ljava/nio/file/Path;)V
 			ARG 1 path
 		METHOD equals (Ljava/lang/Object;)Z
 			ARG 1 o
 		METHOD m_awnmrrfa getName ()Ljava/lang/String;
-		METHOD m_bvgoafsy getOldLevelDatPath ()Ljava/nio/file/Path;
+		METHOD m_bvgoafsy getOldWorldDatPath ()Ljava/nio/file/Path;
```

Comment:
Is "Dat" short for "Data"? If so, I think it's worth the extra letter for the clarity to just have "Data".

Same for the next two methods ofc.

Reply:
some parts of minecraft worlds are saved in a `.dat` format, and this is the path to that file

Reply:
ah ok